ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc

```diff
@@ -1656,6 +1779,134 @@ static GstTensorFilterFramework NNS_support_tensorflow_lite
               .allocateInInvoke = nullptr,
           } } };
 
+#if defined(__ANDROID__)
+/**
+ * @brief Set additional environment (ADSP_LIBRARY_PATH) for tflite
+ */
+static gboolean
+_tflite_set_env (JNIEnv *env, jobject context)
```

Comment:
Name this ```_tflite_set_env_android``` for better readability.
Then, separate conditions for "QNN" and "ANDROID". Don't use the "__ANDROID__" as if that is same with "QNN". You should be able to build Android without QNN without touching any .cc or .h codes.

Reply:
I added a function `_tflite_set_env_qnn_android`, PTAL